server/components/auth/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by sourabh on 26/7/17.
+ */
+const jwt_token = require("jsonwebtoken");
+const compose = require("compose-middleware").compose;
+const verifyTsmsToken = (req, res, next) => {
+  //to verify token
+  const token = req.headers.authorization;
+  if (token) {
+    jwt_token.verify(token, process.env.SECRET_KEY, (err, decode) => {
+      if (err) {
+        res.status(500).send("invalid token");
+      }
+      else {
+        const decodeData = jwt_token.decode(token);
+        req.ememployeeId = decodeData.employeeId;
+        next();
+      }
+    })
+  } else {
+    next();
+  }
+};
+
+const authChecker = (req, res, next) => {
```

Comment:
Add documentation for function:
/**
* <Purpose of function>
* @params - <parameters>
* @return - <return object>
*/